### ISSUE TITLE
Python 3.9 setup.py: open("README.rst", "U") ValueError: invalid mode: 'U'

### ISSUE THREAD

**Issue by hroncok** [wolever/parameterized#90]
I get this error in Python 3.9.0a2:

```
$ python3 setup.py build
Traceback (most recent call last):
  File "/builddir/build/BUILD/parameterized-0.7.0/setup.py", line 11, in <module>
    long_description = open("README.rst", "U").read()
ValueError: invalid mode: 'U'
```

(That is 0.7.0, but the code is there on master as well.)

The U mode was removed in Python 3.9: https://bugs.python.org/issue37330

**Comment by vstinner** [comment]
I proposed https://github.com/wolever/parameterized/pull/94 to port parameterized to Python 3.9.

**Comment by vstinner** [comment]
Fedora downstream issue: https://bugzilla.redhat.com/show_bug.cgi?id=1787948

**Comment by wolever** [comment]
This has been fixed on 0.7.3. Thanks for the Fedora port!
